jenkins/automation/rgw/cortx-rgw-main.groovy

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env groovy
```

Comment:
For main branch update - https://github.com/Seagate/cortx-re/blob/main/jenkins/internal-ci/generic/branch/cortx-rgw.groovy

Reply:
Done